src/main/resources/assets/engineersdecor/models/block/furniture/steel_catwalk_ta4_model.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
I was thinking about creating a custom `BakedModel` which will just merge the two models and apply the necessary transformations, but that seemed unnecessary for just one block, maybe if you decide to do a big refactor in the future

Reply:
Hi, no I also think simplicity is the best approach. Models will anyway be in RAM at the end of the day, if generated and cached or generated by vanilla at start up. Like this it's likely also better maintainable.

Reply:
The main idea behind that is if one of the models gets changed for whatever reason, the change will be reflected on the merged model as well, which would feel adequate considering the nature of the block. One could argue this also leads to better maintainability. But again, probably not worth just for a single block.